lib/pg_query/deparse.rb

```diff
@@ -22,7 +22,7 @@ def from(item)
 
     def deparse_item(item, context = nil) # rubocop:disable Metrics/CyclomaticComplexity
       return if item.nil?
-      return item if item.is_a?(Fixnum)
+      return item if item.is_a?(Integer)
```

Comment:
I think this works with any version rubies.

- In ruby 2.3.x or older, `Integer` class is an ancestor of `Fixnum` and `Bignum`.
- In ruby 2.4+, all integers are `Integer` class.

So `item.is_a?(Integer)` works in both cases.
Supporting `Bignum` in older rubies is a difference, but it may work as intended.